Nakama.Tests/Socket/WebSocketTest.cs

```diff
@@ -65,6 +66,7 @@ public async Task ShouldCreateSocketAndDisconnect()
             _socket.Closed += () => completer.SetResult(true);
 
             await _socket.ConnectAsync(session);
+            await Task.Delay(1000);
             await _socket.CloseAsync();
```

Comment:
What was failing here?

Reply:
Without the delay the test times out, seems like the `_socket.Closed` event is never received.

Reply:
Hmm, could you keep the test failing so we know to investigate it?